L1 scoring

sohnshaik opened the pull request:
l1 scoring

Commit:
unfinish code sequence but i needed to check smthing on a diff branch

Commit:
Revert "unfinish code sequence but i needed to check smthing on a diff branch"

This reverts commit e17c3dbe86bcc895158378f8a68d0abb7f28d0c1.

Commit:
coral cmd

Commit:
finger cmds written

Commit:
l1 works yippee

fortinj1354 reviewed (dismissed):
Untested

sohnshaik committed:
l1 faster scoring

philosophermurr reviewed (approved):
great work guys woohoo yeah i reviewed this on my phone yeah awesome yippee

sohnshaik committed:
L1 additions

theoneandonlyr3n committed:
L1 scoring improved

banks approved, focus on scoring on the left corner of the reefs - lowered ele height and made slowScore faster
